Storybook: Add vision statement to intro page

jessover9000 opened the pull request:
Added the vision statement compiled by the Grafana Labs design system knights to the grafana-ui intro section. The vision statement gives a clearer idea why grafana-ui exists and how it can serve our users. Feedback and further improvement ideas welcome!

jessover9000 committed:
Storybook: Add vision statement to intro page

Added the vision statement compiled by the Grafana Labs design system knights to the grafana-ui intro section. Feedback and further improvement ideas welcome!

oddlittlebird commented on packages/grafana-ui/src/Intro.story.mdx:
```diff
@@ -1,12 +1,26 @@
+
+**Our vision**
+Grafana Labs started grafana-ui to make contributing to Grafana as easy as possible for Grafanistas and community members of all fields. We want to create a component library that results in
+1. Understanding of how each component works and how you can use it to create a great user experience. 
```
This should be an unordered (bulleted) list.

oddlittlebird commented on packages/grafana-ui/src/Intro.story.mdx:
```diff
@@ -1,12 +1,26 @@
+## A library of unified components and styles
+Grafana-ui is a library of reusable components and guidelines on how to use and develop them. You can arrange them into any complex interface you need, for example in your Grafana plugin, streamlining your design process.
+
+**Our vision**
```
Please turn the bold subheadings in this section into h3s.

oddlittlebird reviewed (approved):
Great stuff. Just a couple minor format things. Please coordinate with @marcusolsson regarding crosslinking from plugin developer docs.

kaydelaney reviewed (approved):
LGTM!

Clarity-89 commented on packages/grafana-ui/src/Intro.story.mdx:
```diff
@@ -1,12 +1,26 @@
 
-If you're a plugin developer, you can use these component to give your plugin an official look-and-feel.
+## A library of unified components and styles
+Grafana-ui is a library of reusable components and guidelines on how to use and develop them. You can arrange them into any complex interface you need, for example in your Grafana plugin, streamlining your design process.
```
I think this paragraph basically repeats the previous one and maybe could be removed.

Clarity-89 commented on packages/grafana-ui/src/Intro.story.mdx:
```diff
@@ -1,12 +1,26 @@
+Grafana-ui is a library of reusable components and guidelines on how to use and develop them. You can arrange them into any complex interface you need, for example in your Grafana plugin, streamlining your design process.
+
+**Our vision**
+Grafana Labs started grafana-ui to make contributing to Grafana as easy as possible for Grafanistas and community members of all fields. We want to create a component library that results in
```
```suggestion
Grafana Labs started grafana-ui to make contributing to Grafana as easy as possible for Grafanistas and community members of all fields. We want to create a component library that results in:
```

Clarity-89 commented on packages/grafana-ui/src/Intro.story.mdx:
```diff
@@ -1,12 +1,26 @@
+4. Transparency about how we work and what we do.
+
+**Maintained by Grafana Labs and you**
+Grafana Labs has a task force that helps create and maintain components. We make sure that components are documented and easy to use. The current status of the grafana-ui development is available on [GitHub](https://github.com/grafana/grafana/projects/26). Feel free to contribute!
```
Should it be `@grafana-ui` throughout the text, the same as it is in the first paragraph?

Clarity-89 commented on packages/grafana-ui/src/Intro.story.mdx:
```diff
@@ -1,12 +1,26 @@
+Grafana Labs has a task force that helps create and maintain components. We make sure that components are documented and easy to use. The current status of the grafana-ui development is available on [GitHub](https://github.com/grafana/grafana/projects/26). Feel free to contribute!
+
+**How to get involved**
+When we notice that we need to change something, we ideate on what the change should be, then we put the change in place and communicate it publicly. Developers and designers create and improve grafana-ui together. Throughout the process, we strive to involve you and meet your needs. We are looking forward to discussing your design and improvement ideas on [GitHub](https://github.com/grafana/graf ...
```
Not sure if it's just me but `ideate` sounds a bit weird. I'd use smth like `brainstorm`.

jessover9000 committed:
Review feedback part 1

_fixed headline formatting
_removed numbered list
_removed paragraph "A library of unified components and styles"
_renamed all occurrences to @grafana/ui
_replaced "ideate" with "determine together"
